app/routers/guest/meta.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any
+
+from fastapi import APIRouter, HTTPException, status
+
+from app import models
+from app.i18n import _
+
+router = APIRouter()
+
+
+@router.get(
+    "/meta/{domain}",
+    tags=["meta"],
+)
+async def get_settings_by_domain(domain: str) -> list[dict[str, str | Any]]:
```

Comment:
If `Any` is needed (I don't think so?) then there is no point in Union'ing it with other types, since Any matches them already.

```suggestion
async def get_settings_by_domain(domain: str) -> list[dict[str, str]]:
```